docs/eventing/samples/apache-camel-source/README.md

```diff
@@ -93,19 +98,44 @@ timer CamelSource from the namespace:
 kubectl delete camelsource camel-timer-source
 ```
 
-Install the [`source_telegram.yaml`](source_telegram.yaml) resource:
+Install the [telegram CamelSource](source_telegram.yaml) from source:
+
+```shell
+ko apply -f source_telegram.yaml
+```
+
+Start kail again and keep it open on the event display:
 
 ```shell
-kubectl apply -f source_telegram.yaml
+kail -d camel-event-display --since=10m
 ```
 
-Now, you can **send messages to your bot** with any Telegram client.
+Now, you can contact your bot with any Telegram client. Each message you send
+to the bot will be printed by the event display as a cloudevent.
+
+
+### Run a Camel K Source
+
+For complex use cases that require multiple steps to be executed before event data is ready to be published, you can use Camel K sources.
+Camel K lets you use Camel DSL to design one or more routes that can define arbitrarily complex workflows before sending events to the target sink.
+
+If you have previously deployed other CamelSources, to reduce noise in the event display, you can remove them all from the namespace:
+
+```shell
+kubectl delete camelsource --all
+```
+
+Install the [Camel K Source](source_camel_k.yaml) from source:
```

Comment:
This needs to be provided in the same dir as this doc

Reply:
You might want to call this something other than "Camel K Source" -- it confused me with the source CRD installed on line 20.